calico-enterprise_versioned_docs/version-3.17/_includes/release-notes/_v3.17.1-release-notes.mdx

```diff
@@ -0,0 +1,36 @@
+08 August 2023
+
+### What's new {#whats-new-3.17.1}
+
+#### New policy recommendations engine for namespace isolation
+{{prodname}} has added a new policy recommendations engine that automatically generates staged policies for namespace isolation within your cluster. [Policy recommendations](network-policy/recommendations/policy-recommendations.mdx).
```

Comment:
I believe `mdx` files are transpiled to web pages. You need to link to `network-policy/recommendations/policy-recommendations` (not `.mdx`). Please fix all other `mdx` references in this release notes.

Reply:
Fixed.